lms/templates/courseware/gradebook.html

```diff
@@ -118,7 +118,17 @@
         </tbody>
       </table>
     </div>
+    <span class="grade-book-footer">
+        %if page["previous_offset"] is not None:
+          <a href="${page_url}?offset=${page['previous_offset']}">&#12296;</a>
```

Comment:
Why are using a code instead of our icon set?

On Fri, Nov 6, 2015 at 9:07 AM, Amir Qayyum Khan <notifications@github.com>
wrote:

> In lms/templates/courseware/gradebook.html
> <https://github.com/edx/edx-platform/pull/10250#discussion_r44139105>:
>
> > @@ -118,7 +118,17 @@
> >          </tbody>
> >        </table>
> >      </div>
> > +    <span class="grade-book-footer">
> > +        %if page["previous_offset"] is not None:
> > +          <a href="${page_url}?offset=${page['previous_offset']}">&#12296;</a>
>
> FYI &#12296; this is code for left angular bracket. I also but aria-label
> in the link
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/edx/edx-platform/pull/10250/files#r44139105>.
>